src/topp/IDFileConverter.cpp

```diff
@@ -329,6 +330,20 @@ class TOPPIDFileConverter :
       else if (in_type == FileTypes::IDXML)
       {
         IdXMLFile().load(in, protein_identifications, peptide_identifications);
+        // get spectrum_references from the mz data, if necessary:
+        if (!mz_file.empty())
```

Comment:
I think so. The spectrum references would need to be read from the `mz_file`, correct?